packages/notifications/push/amplify_push_notifications/ios/Classes/PushNotificationsNativePlugin.h

```diff
@@ -1,7 +1,7 @@
 // 
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-// Autogenerated from Pigeon (v7.2.1), do not edit directly.
+// Autogenerated from Pigeon (v9.2.4), do not edit directly.
```

Comment:
These too. Again, out of scope for the PR. Just making a note. We put these in a `pigeons` dir for some of the other packages so that we could treat anything in that dir as generated.

```
**/pigeons/*.h linguist-generated=true
**/pigeons/*.m linguist-generated=true
```